nowcasting_dataset/data_sources/gsp/gsp_model.py

```diff
@@ -11,7 +11,7 @@ class GSP(DataSourceOutput):
 
     __slots__ = ()
     _expected_dimensions = ("time", "id")
-    _expected_data_vars = ("data", "pv_system_row_number", "x_coords", "y_coords")
+    _expected_data_vars = ("data", "capacity_mwh", "pv_system_row_number", "x_coords", "y_coords")
```

Comment:
do you think "data" should be change to "power_mw" to make it really clear, @JackKelly @jacobbieker